server/src/main/resources/org/opensearch/bootstrap/test-framework.policy

```diff
@@ -157,4 +157,6 @@ grant {
   permission java.lang.RuntimePermission "reflectionFactoryAccess";
   permission java.lang.RuntimePermission "accessClassInPackage.sun.reflect";
   permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
+  permission java.util.PropertyPermission "opensearch.set.netty.runtime.available.processors", "write";
+  permission java.net.SocketPermission "*", "accept,connect";
```

Comment:
Could you please explain why this second one is needed?

Reply:
this second one is useless now, I will delete it.